### DISCUSSION TITLE
[Feature] Album Addition Dialog Box with Multiselect

### DISCUSSION THREAD

**Discussion by murtaza-nasir** [immich-app/immich#12540] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I would like to request the addition of checkboxes, so that multiple albums can be selected in the album addition dialog box. 

![365569316-3d595bb0-cc80-4a9a-8d6f-1e436de8fc1b](https://github.com/user-attachments/assets/df80c5ce-6ca6-4e38-8cbd-15b7351fd7fc)

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#11259
